### PR TITLE
Sample App:Demonstration of sine curve

### PR DESCRIPTION
**Pull request by farheen21**
As per the instruction of Andrew Brogdon redbrogdon@gmail.com,I have build an app for GSOC 2020 , which demonstrate how to implement sine curve in flutter with its graph.kindly give it look and give your feedback.

Please go throught [How to contribute guide](https://github.com/divyanshub024/awesome-dart-pad/blob/master/Contribution.md) before creating a PR.

### CODE REVIEW

**Comment by divyanshub024** [comment]
@farheen21 there are some conflicts. Can you resolve them?

**Comment by farheen21** [comment]
@divyanshub024 Sorry for getting the conflicts, I have resolve them and add the gif file also.Kindly check it again and tell me is there any other change you want.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+
+
+=======
```
Comment (divyanshub024) [inline README.md]: Can you remove all these 3 lines.

**Review by divyanshub024 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (farheen21) [reply README.md]: @divyanshub024 Hey! as per your instruction I removed them.

**Review by divyanshub024 — APPROVED** [review]
Thanks a lot, @farheen21!! Great sample!!

**Pull request merged**